x-pack/plugins/rollup/public/crud_app/sections/job_create/job_create.js

```diff
@@ -239,8 +239,22 @@ export class JobCreateUi extends Component {
         return;
       }
 
-      // TODO: Show toast or inline error.
+      const { error: errorString, statusCode } = error.data;
```

Comment:
any concerns about if `error` or `error.data` is undefined/not an object?

Reply:
Good point, fixed!